src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1207,32 +1208,119 @@ void missingRdfTypeOnNamedNode() {
         }
     }
 
+    class BaseExample {
```

Comment:
I would add a java doc or comment here describing the purpose of this class and what is used for.

Reply:
This class has been removed as part of [`d72c1`](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/pull/134/commits/d72c1df86a351b1aed6e80ad7a548e69ea454427), and its functionality moved to the `ExamplesAndValidationTest` class.